modules/component/src/component.module.ts

```diff
@@ -0,0 +1,13 @@
+import { NgModule } from '@angular/core';
+import { LetDirective } from './let';
+import { PushPipe } from './push';
+
+export const DECLARATIONS = [LetDirective, PushPipe];
+export const EXPORTS = [DECLARATIONS];
+
+@NgModule({
+  declarations: [DECLARATIONS],
+  imports: [],
+  exports: [EXPORTS],
+})
+export class ComponentModule {}
```

Comment:
As this name is probably being used pretty commonly, we should change this to something like `ReactiveComponentModule`

Reply:
regarding prefix:
https://github.com/ngrx/platform/pull/2046#discussion_r373784703

Reply:
@brandonroberts 
> As this name is probably being used pretty commonly, we should change this to something like ReactiveComponentModule

It will also affect directives, pipes, and services IMHO. Maybe more general: `ReactiveFeaturesModule`?

Reply:
Changed to ReactiveComponentModule:
https://github.com/ngrx/platform/pull/2046/commits/d52467f8a17e784108ce2a0bcc36feacd692a277